code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -30,6 +30,8 @@
 	var/target_temperature = T20C
 	var/heat_capacity = 0
 	var/interactive = TRUE // So mapmakers can disable interaction.
+	var/allows_cooling = TRUE // Non anomalous thermomachines are always allowed to cool
```

Comment:
1. Make this a doc comment in the form of:

```dm
/// Three slashes
var/var_declaration_goes_here
```

2. This comment doesn't explain what this does, just what sets it

Reply:
Documented not just allows_cooling but also has_pyro_anomaly_core because I wasn't sure if it was also needed to be documented.